index.html

```diff
@@ -2,232 +2,239 @@
+          <h1 class="logo pop" style="font-size: 2.5rem !important">The XR Social Music Platform</h1>
+          <h2 class="motto">Easily create, share and enjoy immersive worlds together</h2>
+          <div class="subscribe">
+            <h5 class="info-text">
+              Join the waiting list for the beta. We'll keep you posted.
+            </h5>
+            <div>
+              <form class="form-inline validate" role="form" action="https://virtision.us1.list-manage.com/subscribe/post?u=9ab8c2f01a3f5aa677094fc1e&amp;id=4c5085e68c" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form"
+                target="_blank" novalidate>
+                <div class="row">
+                  <div class="form-group">
+                    <input type="email" name="EMAIL" class="form-control transparent" placeholder="Your email here...">
+                    <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_9ab8c2f01a3f5aa677094fc1e_4c5085e68c" tabindex="-1" value=""></div>
+                  </div>
+                  <button type="submit" class="btn btn-first btn-email btn-fill">NOTIFY ME</button>
+                </div>
+              </form>
+
+              <div class="media mb-3">
+                <a href="https://www.linkedin.com/company/virtision-com">
+                  <i class="fa fa-linkedin"></i>
+                </a>
+
+                <a href="https://www.facebook.com/virtision">
+                  <i class="fa fa-facebook"></i>
+                </a>
+
+                <a href="https://www.instagram.com/virtision/">
+                  <i class="fa fa-instagram"></i>
+                </a>
+
+              </div>
+            </div>
+          </div>
+        </div>
+        <div class="col-lg-5 offset-lg-1 float-right mb-3">
+          <video playsinline="" muted="" class="rounded shadow-lg video" src="images/demo.mp4" loop autoplay></video>
+        </div>
+
+      </div>
+    </div>
+  </div>
+  <div class="main">
+    <div class="container value">
+      <div class="row justify-content-center">
+        <div class="col-lg-8 center">
+          <h1 class="pop"> For any creators at any level to create and share their imagination in XR</h1>
+          <!--Toy Faces created by Amrit Pal Singh (https://amritpaldesign.com/toy-faces)-->
+        </div>
+        <div class="col-lg-12">
+          <img style="width:inherit" class="rounded shadow-lg" src="./images/artist2.png">
+        </div>
+
+      </div>
+    </div>
+    <div class="container value">
+      <div class="row pt-5">
+        <div class="col-lg-6">
+          <img class="demo" src="./images/creation.png" width="400">
+        </div>
+        <div class="col-lg-6">
+          <h1 class="pop">Simply create your XR story with just a few taps</h1>
+          <h3 class="motto">Our seamless creation experience will unblock your creativity.</h3>
```

Comment:
"... will unblock your creativity." sounds too much like PM-speak.

How about: 
a) "Our seamless creation experience keeps your ideas flowing."
b) "... lets you focus on building your world." 

or similar..

----

Future suggestion - we should add a GIF here to highlight how few taps it can be.

![image](https://user-images.githubusercontent.com/8660050/117747266-ae3df580-b1c2-11eb-889e-fe06381d59dd.png)